MroongaのDockerイメージ更新が原因でテストに失敗する

以下のコミットでMroongaのDockerイメージが更新されてから、テスト時のデータベース作成で失敗するようになったらしい。

https://github.com/mroonga/docker/commit/3560699d3b726bb6ad42707a4131de83f8556c46

ローカル環境で `docker logs` を実行すると、以下が出力された。

```
2021-06-13 03:18:11+00:00 [ERROR] [Entrypoint]: Database is uninitialized and password option is not specified
    You need to specify one of the following:
    - MYSQL_ROOT_PASSWORD
    - MYSQL_ALLOW_EMPTY_PASSWORD
    - MYSQL_RANDOM_ROOT_PASSWORD
```

CI用なので、とりあえず環境変数 `MYSQL_ALLOW_EMPTY_PASSWORD` を指定して、データベースを作成できるようにする。

--- a/.github/workflows/coverage.yaml
+++ b/.github/workflows/coverage.yaml
@@ -20,6 +20,8 @@ jobs:
     services:
       db:
         image: groonga/mroonga:latest
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: 1
         ports:
           - 33060:3306
       redis:

--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -27,6 +27,8 @@ jobs:
     services:
       db:
         image: groonga/mroonga:${{ matrix.mroonga }}
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: 1
         ports:
           - 33060:3306
       redis:

